### PR TITLE
Alert component

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Made alert component.
There are six types of alerts.
1. alert-primary
2. alert-secondary
3. alert-success
4. alert-danger
5. alert-warning
6. alert-light

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.alert{
+    border: 2px solid;
+    padding: 1rem;
+    margin: 1rem 0;
+    font-size: 1.5rem;
+    border-radius: 0.5rem;
+}
+.alert-primary{
+    color: #004085;
```
Comment (vishal804) [inline components/alert.css]: Try to use Color Variable

Comment (arjun-computer-geek) [reply components/alert.css]: fixed it

**Review by vishal804 — APPROVED** [review]

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.alert{
+    border: 2px solid;
+    padding: 1rem;
+    margin: 1rem 0;
+    font-size: 1.5rem;
+    border-radius: 0.5rem;
+}
+.alert-primary{
```
Comment (junaidshaikh-js) [inline components/alert.css]: Keep a blank line between two blocks. Use code formatter.

**Review by shivanigangadharan — APPROVED** [review]
Looks good. 👍

File: documentation/docs.html [hunk 3]
```diff
@@ -92,6 +92,48 @@ <h3 class="small-title">Example of Image Avatar</h3>
...
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+
+            </div>
+        </div>
+
     </main>
     <!-- Main -- Ends -->
     <script src="../index.js"></script>
```
Comment (junaidshaikh-js) [inline documentation/docs.html]: add `defer` attribute to script tag.

File: components/alert.css (hunk 2 above)
Comment (arjun-computer-geek) [reply components/alert.css]: ok i will use

Comment (arjun-computer-geek) [reply components/alert.css]: fixed it

File: documentation/docs.html (hunk 3 above)
Comment (arjun-computer-geek) [reply documentation/docs.html]: ok thanks man

Comment (arjun-computer-geek) [reply documentation/docs.html]: done bro

**Review by Shubhamkr2610 — APPROVED** [review]
Great work , your  code is clean and readable . Looks good to me .

**Pull request merged**